andela/ah-infinity-stones-frontend
# #161966881 Implement article tags

#### What does this PR do?
- Implement article tags

#### Description of Task to be completed?
- Add tags in all relevant displays

#### How should this be manually tested?
1. `$ git clone -b ft-tags-161966881 https://github.com/andela/ah-infinity-stones-frontend.git`  and `$ cd ah-infinity-stones-frontend`
2. Install the dependencies - `$ npm install`
3. Start the app by running `$ npm start` - This will run the app in the development mode.  
 To test the feature. Open http://localhost:3000/login  to view it in the browser. Provide email and password and click submit. This assumes that you have visited `http://localhost:3000/signup`, created and activated your account.
4. Visit http://localhost:3000 , and open an article. The article tag is displayed at the bottom of the article and on click it redirects to articles that contain that tag.
4. Test the app by running `$ npm test` - This will Launch the test runner in the interactive watch mode.

#### What are the relevant pivotal tracker stories?
[#161966881](https://www.pivotaltracker.com/story/show/161966881)

#### Screenshots
![image](https://user-images.githubusercontent.com/17976289/51801070-0d71d900-224a-11e9-988d-31cd8902badc.png)
![image](https://user-images.githubusercontent.com/17976289/51801085-32664c00-224a-11e9-9a71-c55942934bbd.png)
